crates/bevy_app/src/plugin_group.rs

```diff
@@ -139,6 +139,21 @@ impl PluginGroupBuilder {
     }
 }
 
+/// A plugin group which doesn't do anything. Useful for examples:
+/// ```rust
+/// # use bevy_app::prelude::*;
+/// use bevy_app::NoopPluginGroup as MinimalPlugins;
+///
+/// fn main(){
+///     App::new().add_plugins(MinimalPlugins).run();
+/// }
+/// ```
+pub struct NoopPluginGroup;
```

Comment:
should this be doc hidden?

Reply:
I suppose so - it's only really useful for dependencies of bevy_internal